src/components/Tooltip/index.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { ReactNode } from "react"
+import {
+  Box,
+  Center,
+  Popover,
+  PopoverArrow,
+  PopoverBody,
+  PopoverContent,
+  PopoverTrigger,
+} from "@chakra-ui/react"
+
+export interface IProps {
+  content: ReactNode
+  children?: ReactNode
+}
+
+const Tooltip: React.FC<IProps> = ({ content, children }) => {
+  return (
+    <Box title="More Info!" display="inline-block" ml={2}>
```

Comment:
We've removed this title from the tooltip
```suggestion
    <Box display="inline-block" ml={2}>
```